internal/pkg/aws/ecs/task.go

```diff
@@ -129,13 +155,17 @@ type TaskStatus struct {
 	CapacityProvider string    `json:"capacityProvider"`
 }
 
-// StoppedTaskStatus contains the status info a stopped task.
+// StoppedTaskStatus contains the status info of a stopped task.
 type StoppedTaskStatus TaskStatus
 
 // HumanString returns the stringified TaskStatus struct with human readable format.
 // Example output:
 //   6ca7a60d          f884127d            RUNNING             19 hours ago       -              UNKNOWN
 func (t TaskStatus) HumanString() string {
```

Comment:
I totally agree! This makes a huge amount of sense since currently these `HumanString()` functions are so customized for `svc status` that they are not necessarily a good string representation of the struct itself.